cli/cmd/stat.go

```diff
@@ -56,12 +56,12 @@ The optional [TARGET] option can be either a name for a deployment or pod resour
 				return fmt.Errorf("invalid resource type %s, only %v are allowed as resource types", friendlyNameForResourceType, []string{k8s.KubernetesPods, k8s.KubernetesDeployments, ConduitPaths})
 			}
 		}
-		kubeApi, err := k8s.NewK8sAPi(shell.NewUnixShell(), kubeconfigPath, apiAddr)
+		kubeApi, err := k8s.NewK8sAPI(shell.NewUnixShell(), kubeconfigPath, apiAddr)
 		if err != nil {
 			return err
 		}
 
-		client, err := newApiClient(kubeApi)
+		client, err := newPublicAPIClient(kubeApi, apiAddr)
```

Comment:
Following up from the comment above, if you rewrite `newPublicAPIClient` as suggested, then you don't need to call `k8s.NewK8sAPI` here (or in any of the other command methods). You can just replace lines 59-64 with:

```
		client, err := newPublicAPIClient()
```